src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      data-qa="big"
+      method="post"
+    >
+      <label>
+        <input
+          type="text"
+          class="form-search search-big"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+
+        <img
+          src="./images/Search.svg"
+          alt="Icon"
+          class="mglass-big"
+        >
+      </label>
+    </form>
+
+    <form
+      action="/"
+      data-qa="small"
+      method="post"
+    >
+      <label>
+        <input
+          type="text"
+          class="form-search search-small"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+
+        <img
+          src="./images/Search.svg"
+          alt="Icon"
+          class="mglass-small"
+        >
+      </label>
```

Comment:
```suggestion
      <label>
        <input
          type="text"
          class="form-search"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >

        <img
          src="./images/Search.svg"
          alt="Icon"
          class="mglass"
        >
      </label>
    </form>

    <form
      action="/"
      data-qa="small"
      method="post"
    >
      <label>
        <input
          type="text"
          class="form-search search-small"
          placeholder="Try &quot;Los Angeles&quot;"
        >

        <img
          src="./images/Search.svg"
          alt="Icon"
          class="mglass mglass-small"
        >
      </label>
```
lets make it work like this